commit 9aa66123be8b8f9ed6246f05e778c873623c0c17
Author: Andrew Hsieh <andrewhsieh@google.com>
Date:   Mon Apr 14 21:40:20 2014 -0700

    [4.9] Add additional multilib option: mfloat-abi=hard
    
    See 5be615df32ce970fcccda93cf577e9ce35b1d397
    
    Change-Id: Iabbfcf0080d966e809e1a8521e93de7fcabd03a4

diff --git a/gcc/config/arm/t-linux-androideabi b/gcc/config/arm/t-linux-androideabi
index 8f1307c..cbbec5b 100644
--- a/gcc/config/arm/t-linux-androideabi
+++ b/gcc/config/arm/t-linux-androideabi
@@ -1,8 +1,9 @@
-MULTILIB_OPTIONS     = march=armv7-a mthumb
-MULTILIB_DIRNAMES    = armv7-a thumb
-MULTILIB_EXCEPTIONS  =
+MULTILIB_OPTIONS     = march=armv7-a mthumb mfloat-abi=hard
+MULTILIB_DIRNAMES    = armv7-a thumb hard
+MULTILIB_EXCEPTIONS  = mfloat-abi=hard* mthumb/mfloat-abi=hard*
 MULTILIB_MATCHES     =
 MULTILIB_OSDIRNAMES  =
+MULTILIB_EXTRA_OPTS  = Wl,--no-warn-mismatch
 
 # The "special" multilib can be used to build native applications for Android,
 # as opposed to native shared libraries that are then called via JNI.
